.pre-commit-config.yaml

```diff
@@ -15,35 +16,50 @@ repos:
       - id: name-tests-test
       - id: check-ast
       - id: debug-statements
+
   - repo: https://github.com/pre-commit/mirrors-mypy
     rev: v1.0.1
     hooks:
       - id: mypy
+
   - repo: https://github.com/asottile/add-trailing-comma
     rev: v2.4.0
     hooks:
       - id: add-trailing-comma
+
   - repo: https://github.com/Lucas-C/pre-commit-hooks
     rev: v1.4.2
     hooks:
       - id: remove-tabs
+
   - repo: https://github.com/pycqa/pydocstyle
     rev: 6.3.0
     hooks:
       - id: pydocstyle
         exclude: ^tests/.*
-  - repo: https://github.com/PyCQA/pylint
```

Comment:
Reconfiguring unrelated linters isn't related to integrating yamllint. Formatting changes must always be separate from the functional ones.

Reply:
As far as I understand, I need to separate one PR for integrate `yamlint` and formatting changes can refer to this PR. The second PR should refer to fix `pylint` and `flake8` in `pre-commit tool`?

Reply:
Also, I have question about `pylint` in CI job. Firstly, I tried to use `.pylintrc`, but have errors with this config, then I looked another implementation in repo of Marta. But, in your repo `cheroot`  I see this config (`.pylintrc`). So, what the significant difference in these 2 implementations?

Reply:
> As far as I understand, I need to separate one PR for integration `yamlint` and formatting changes can refer to this PR.

In if the formatting changes are directly related to the yamllint configuration, they should be bundled. If they aren't related to linters but are your personal preference, it's probably keep them separate.

Reply:
> The second PR should refer to fix `pylint` and `flake8` in `pre-commit tool`?

Only if these are broken in the same way and have the same root of issue. If they aren't related, there's no reason why one would put them into the same patch. Instead, it'd best to make the changes atomic.